kuma/users/tests/__init__.py

```diff
@@ -27,8 +27,7 @@ def user(save=False, **kwargs):
     password = kwargs.pop('password', 'password')
     user = User(**kwargs)
     user.set_password(password)
-    if save:
-        user.save()
+    user.save()
```

Comment:
Not sure, I think some tests didn't pass without it? I'll try again without that change.